app/models/subnet.rb

```diff
@@ -121,7 +121,19 @@ class Jail < ::Safemode::Jail
   def self.network_reorder(order_string = 'network')
     adapter = connection.adapter_name.downcase
     if adapter.starts_with?('postgresql')
-      reorder(order_string.sub('network', 'inet(network)'))
+      subnet_table = Subnet.arel_table
+      order_by = Arel::Nodes::NamedFunction.new('INET', [subnet_table[:network]])
```

Comment:
This assumes the `order_string` is network, but it doesn't have to be. Also note that previously the API worked with multiple columns as well, but this breaks it.

Reply:
It's always `newtwork` in our case. Multiple columns were already filtered on the controller level and have never worked with this.